app/assets/javascripts/application.js

```diff
@@ -0,0 +1,3 @@
+//= require jquery
+//= require jquery_ujs
```

Comment:
govuk_admin_template provides jQuery so the top line can be removed.
What is `jquery_ujs` used for?